coretex/cli/modules/node.py

```diff
@@ -105,6 +109,44 @@ def registerNode(name: str) -> str:
     return accessToken
 
 
+def selectModelId(retryCount: int = 0) -> int:
+    if retryCount >= 3:
```

Comment:
There is no need for `retryCount` for this, you can remove it. `fetchById` function already has built-in retry mechanism

Reply:
This retry count I've added is copying old (typescript) CLI mechanishm, so if user specifies invalid model id he will have prompt again and if same happens again he will have one more chance to specifiy good model id before terminating CLI.